boards/BMS/Inc/Io/Io_LTC6813/Io_SharedLTC6813.h

```diff
@@ -45,21 +45,21 @@ enum RegGroupByteFormat
  * @param tx_data The command or register group to pack PEC15 bytes
  * @param size The size of the command or register group to transmit
  */
-void Io_NewLTC6813_PackPec15(uint8_t *tx_data, uint8_t size);
+void Io_SharedLTC6813_PackPec15(uint8_t *tx_data, uint8_t size);
 
 /**
  * Prepare the command to be sent to the LTC6813 by packing the command and
  * PEC15 bytes into the tx_cmd buffer
  * @param tx_cmd The buffer to pack the command along with the PEC15 check
  * @param cmd The command to pack into tx_cmd
  */
-void Io_NewLTC6813_PrepareCmd(uint16_t *tx_cmd, uint16_t cmd);
+void Io_SharedLTC6813_PrepareCmd(uint16_t *tx_cmd, uint16_t cmd);
 
 /**
  * Initialize the SPI handle used to communicate with the LTC6813
  * @param spi_handle The given SPI handle for the LTC6813 daisy chain.
  */
-void Io_NewLTC6813_InitSpiHandle(SPI_HandleTypeDef *spi_handle);
+void Io_SharedLTC6813_InitSpiHandle(SPI_HandleTypeDef *spi_handle);
```

Comment:
I think we should put the init function at the top as an indication that it should called on startup

Reply:
hm ya agree